### PR TITLE
fix #196 now when you view styles in the console, they point to the less...

### PR DESCRIPTION
**Pull request by nelsonomuto**
... file

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "nonbsp": true,
     "nonew": true,
     "plusplus": false,
-    "quotmark": "double",
+//    "quotmark": "double",
```
Comment (FezVrasta) [inline .jshintrc:18]: please, no

File: template/material/css/styles.map [hunk 2]
```diff
@@ -0,0 +1 @@
...Ad,kBAhDiC,MAEzC,iBAqCI,MASK,aAAa;AAAd,kBAhDM,OAEI,iBAAgB,aAqC9B,MASK,aAAa;AAAd,kBAhDiC,MAEvB,iBAAgB,aAqC9B,MASK,aAAa;AAAd,kBAhDM,OAEd,iBAqCW,QASF,aAAa;AAAd,kBAhDiC,MAEzC,iBAqCW,QASF,aAAa;AAAd,kBAhDM,OAEI,iBAAgB,aAqCvB,QASF,aAAa;AAAd,kBAhDiC,MAEvB,iBAAgB,aAqCvB,QASF,aAAa;EACV,SAAS,EAAT;;AAEJ,kBAnDM,OAEd,iBAqCI,MAYK;AAAD,kBAnDiC,MAEzC,iBAqCI,MAYK;AAAD,kBAnDM,OAEI,iBAAgB,aAqC9B,MAYK;AAAD,kBAnDiC,MAEvB,iBAAgB,aAqC9B,MAYK;AAAD,kBAnDM,OAEd,iBAqCW,QAYF;AAAD,kBAnDiC,MAEzC,iBAqCW,QAYF;AAAD,kBAnDM,OAEI,iBAAgB,aAqCvB,QAYF;AAAD,kBAnDiC,MAEvB,iBAAgB,aAqCvB,QAYF;EACG,iBAAA;EACA,uBAAA;EACA,SAAA;;AAtDhB,kBAAkB,OA0Dd;AA1DuB,kBAAkB,MA0DzC;EACI,kBAAA;EACA,aAAA;EACA,SAAA;EACA,sBAAA;EACA,kBAAA;EACA,YAAA;EACA,sBAAA;EACA,wEAAA;EACA,kBAAA;EACA,UAAA;EACA,eAAA;;AArER,kBAAkB,OA0Dd,oBAYI;AAtEmB,kBAAkB,MA0DzC,oBAYI;EACI,yBAAA;;AAvEZ,kBAAkB,OA0Dd,oBAeI;AAzEmB,kBAAkB,MA0DzC,oBAeI;EACI,yBAAA;;AA1EZ,kBAAkB,OA0Dd,oBAkBI;AA5EmB,kBAAkB,MA0DzC,oBAkBI;AA5ER,kBAAkB,OA0Dd,oBAkBe,UAAS;AA5ED,kBAAkB,MA0DzC,oBAkBe,UAAS;EAChB,yBAAA;;AA7EZ,kBAAkB,OA0Dd,oBAqBI;AA/EmB,kBAAkB,MA0DzC,oBAqBI;AA/ER,kBAAkB,OA0Dd,oBAqBuB;AA/EA,kBAAkB,MA0DzC,oBAqBuB;EACf,kBAAA;EACA,eAAA;;AAjFZ,kBAAkB,OAoFd,iBAAiB;AApFM,kBAAkB,MAoFzC,iBAAiB;EACb,cAAA"}
```
Comment (FezVrasta) [inline template/material/css/styles.map:1]: why "styles.css" and not the usual file name?

File: .gitignore [hunk 3]
```diff
@@ -8,3 +8,5 @@ Thumbs.db
 .DS_Store
 /node_modules/
 .grunt/
+.idea
```
Comment (SomeoneWeird) [inline .gitignore:11]: whyyyyyyyyyyy

File: Gruntfile.js [hunk 4]
```diff
@@ -8,13 +8,29 @@ module.exports = function(grunt) {
         less: {
             compileless: {
                 options: {
-                    paths: ["less"]
+                    paths: ["less"],
+                    sourceMap: true,
+                    sourceMapFilename: 'dist/css/less.map',
```
Comment (FezVrasta) [inline Gruntfile.js:13]: here would be nice to have the name of the CSS (material.map) and (material-wfont.map)

File: dist/css/material-wfont.css [hunk 5]
```diff
@@ -4732,3 +4586,4 @@ fieldset[disabled] .navbar .btn-link:focus {
 .selectize-control.multi .dropdown-active ~ .selectize-dropdown {
   display: block;
 }
+/*# sourceMappingURL=less.map */
```
Comment (FezVrasta) [inline dist/css/material-wfont.css:4589]: here you have lost all the browser specific prefixes, try to run the tasks of grunt manually

```
grunt less
grunt autoprefixer
grunt cssmin
grunt uglify
grunt clean
grunt copy
```

**Comment by FezVrasta** [comment]
Please check my review, also, please the PR should go to the "develop" branch. thanks for your work!

**Pull request closed without merging**